### DISCUSSION TITLE
Is there a way I can go to a chrome directory like "chrome://settings"?

### DISCUSSION THREAD

**Discussion by ZakDevOnGitHub** [cefsharp/CefSharp#4966] [General]
I know you can go to `chrome://version`, but I also want to know if there's a way I can go to places like `chrome://settings` or `chrome://flags`?

**Comment by amaitland** [comment]
It depends, `Alloy` styled windows (the default) then no. `Chrome` styled windows, which are only supported in `WinForms` and `Wpf.HwndHost` then yes you can use more of the built in pages.

https://github.com/cefsharp/CefSharp/issues/4835#issuecomment-2182373137
